lib/Dialect/Polynomial/IR/PolynomialOps.td

```diff
@@ -340,4 +340,30 @@ def Polynomial_INTTOp : Polynomial_Op<"intt", [Pure]> {
   let hasVerifier = 1;
 }
 
+def Polynomial_EvalOp : Polynomial_Op<"eval"> {
+    let summary = "Evaluate a static polynomial attribute at a given SSA value.";
+    let description = [{
+        Evaluates the result of a polynomial specified as a static attribute at a given SSA value.
+        The result represents the evaluation of the polynomial at the input value and produces
+        a corresponding constant value.
```

Comment:
```suggestion
        a corresponding scalar value.
```

"constant" would imply statically known, and if the input is an SSA value then it's not in general.